packages/core/admin/ee/admin/pages/SettingsPage/pages/AuditLogs/ListView/ComboboxFilter/index.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Combobox, ComboboxOption } from '@strapi/design-system/Combobox';
+import { createGlobalStyle } from 'styled-components';
+
+const GlobalOverflowAuto = createGlobalStyle`
+  /* Prevent scroll on combobox click when the page has an overflow */
+  html, body {
+    overflow: auto;
+  }
+`;
+
+const ComboboxFilter = ({ value, options, onChange }) => {
+  return (
+    <>
+      <GlobalOverflowAuto />
```

Comment:
Is there a way to prevent that? Is that an issue that stems from the DS? This should really be the last resort 🙈

Reply:
Yeah I know this isn't great. I tried a [few other things](https://github.com/strapi/strapi/pull/15504#discussion_r1088117140) but no luck. I'll keep digging in the DS to see if I can resolve the issue over there

Reply:
Maybe @joshuaellis has an idea here as the DS 🧙🏼 ?

Reply:
I think this is a DS issue no? https://github.com/strapi/design-system/issues/818

I haven't had time to look at it... but imo we should tackle the issue where it is, otherwise _everyones_ gonna have to do this solution.

Reply:
gotcha then I'll investigate the issue further over there

Reply:
Thanks Mark! :)